Dockerfile

```
@@ -277,20 +277,20 @@ RUN make distclean || echo "nothing to clean"
 RUN ./configure $GRASS_CONFIG \
     && make -j $NUMTHREADS \
     && make install && ldconfig \
-    &&  cp /usr/local/grass84/gui/wxpython/xml/module_items.xml module_items.xml; \
-    rm -rf /usr/local/grass84/demolocation; \
-    rm -rf /usr/local/grass84/fonts; \
-    rm -rf /usr/local/grass84/gui; \
-    rm -rf /usr/local/grass84/share; \
-    mkdir -p /usr/local/grass84/gui/wxpython/xml/; \
-    mv module_items.xml /usr/local/grass84/gui/wxpython/xml/module_items.xml;
+    &&  cp /usr/local/grass85/gui/wxpython/xml/module_items.xml module_items.xml; \
+    rm -rf /usr/local/grass85/demolocation; \
+    rm -rf /usr/local/grass85/fonts; \
+    rm -rf /usr/local/grass85/gui; \
+    rm -rf /usr/local/grass85/share; \
+    mkdir -p /usr/local/grass85/gui/wxpython/xml/; \
+    mv module_items.xml /usr/local/grass85/gui/wxpython/xml/module_items.xml;
```

```suggestion
    &&  cp /usr/local/grass8?/gui/wxpython/xml/module_items.xml module_items.xml; \
    rm -rf /usr/local/grass8?/demolocation; \
    rm -rf /usr/local/grass8?/fonts; \
    rm -rf /usr/local/grass8?/gui; \
    rm -rf /usr/local/grass8?/share; \
    mkdir -p /usr/local/grass8?/gui/wxpython/xml/; \
    mv module_items.xml /usr/local/grass8?/gui/wxpython/xml/module_items.xml;
```

... maybe even easier?

---

Or we do that elsewhere.

---

It is possible to use a wildcard when creating a directory with mkdir?

---

> Or we do that elsewhere.

Let's add this to a small todo for this week